src/Modules/SimplCommerce.Module.Core/wwwroot/admin/core.module.js

```diff
@@ -64,6 +64,16 @@
                     url: '/countries/edit/:id',
                     templateUrl: 'modules/core/admin/countries/country-form.html',
                     controller: 'CountryFormCtrl as vm'
+                })
+                .state('state-province-create', {
+                    url: '/:countryId/state-province/create',
```

Comment:
I will make the route updates and make a standalone page.

Adding package-lock.json to source control was done inadvertently. I'll add that to .gitignore so that doesn't happen again. However, from reading about the purpose of this file, it says it's meant to be committed; but I'll remove it unless you say otherwise.

Reply:
Adding package-lock.json bring some noise. But it actually is recommended (just read). 
Ok, we should follow the recommendation, add it to source control.
Thanks